### PR TITLE
data: Udayan-Singh

### PR DESCRIPTION
**Pull request by Udayan-Singh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5031"></a>

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]
Looking great 🎉

**Review by imbeshat — APPROVED** [review]

**Review by SurajPratap10 — DISMISSED** [review]

**Review by Sindhuinti — DISMISSED** [review]

**Review by Hrishi5111998 — DISMISSED** [review]

**Review by Krishnapro — DISMISSED** [review]
LGTM 🚀

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by sidharthviz — DISMISSED** [review]

File: data/Udayan-Singh.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Udayan Singh",
+  "type": "personal",
+  "bio": "Tech Undergrad trying to get started with Open Source contributions."
```
Comment (imbeshat) [inline data/Udayan-Singh.json:4]:
```suggestion
  "bio": "Tech Undergrad trying to get started with Open Source contributions.",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**